media/css/_faculty_structure.scss

```diff
@@ -0,0 +1,212 @@
+.bg-structure {
+  position: relative;
+  width: 890px;
+  margin: 0 auto;
+  height: 1010px;
+  padding: 5px;
+  background-image: url('../img/line.png');
+  background-position: top -15px center;
+  background-repeat: no-repeat;
+}
+
+.struct-elem {
+  display: inline-block;
+  position: absolute;
+  background: $grayish-cyan;
+  border: 1px solid $light-black;
+  text-align: center;
+  padding: 9px;
+  font-size: 13px;
+
+  a {
+    color: $white;
+    text-decoration: none;
+    font-size: 1.5em;
+  }
+
+  &:hover {
+    background: $strong-blue;
+  }
+}
+
+.faculty-council {
+  margin-top: 5px;
+  margin-left: 39%;
+  font-size: 16px;
+  text-transform: uppercase;
+}
+
+.dean-item1 {
+  margin-left: 46.7%;
+  margin-top: 45px;
+}
+
+.prof-union {
+  font-size: 13px;
+  margin-left: 603px;
+  margin-top: -21px;
+}
+
+.dean-item2 {
+  width: 300px;
+  margin-left: 100px;
+  margin-top: 40px;
+}
+
+.dean-item3 {
+  margin-left: 490px;
+  margin-top: 40px;
+}
+
+.sno {
+  margin-left: 178px;
+  margin-top: 180px;
+}
+
+.starostat {
+  margin-left: 178px;
+  margin-top: 235px;
+}
+
+.profsouz {
+  margin-left: 625px;
+  margin-top: 235px;
+}
+
+.stydsovet {
+  margin-left: 625px;
+  margin-top: 178px;
+}
+
+.brsm {
+  margin-left: 625px;
+  margin-top: 320px;
+
+  a {
+    font-size: 1.2em;
+  }
+}
+
+.dean-block {
+  margin-top: 60px;
+  width: 100px;
+}
+
+.second-block {
+  .struct-elem {
+    width: 240px;
+  }
+}
+
+.resize-stud-union {
+  .struct-elem {
+    width: 106px;
+  }
+}
+
+.third-block {
+  display: inline-block;
```

Comment:
Properties should be ordered background, border, display, margin-top, padding, position, text-align, width